modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectActionLocalServiceTest.java

```diff
@@ -3043,6 +3075,37 @@ private void _testAddObjectActionWithCircularReference(
 		}
 	}
 
+	private void _testUnbindObjectDefinitions(
+			long objectActionId, ObjectRelationship objectRelationship)
+		throws Exception {
+
+		_objectRelationshipLocalService.updateObjectRelationship(
+			objectRelationship.getExternalReferenceCode(),
+			objectRelationship.getObjectRelationshipId(), 0,
+			objectRelationship.getDeletionType(), false,
+			objectRelationship.getLabelMap(), null);
+
+		ObjectAction objectAction = _objectActionLocalService.fetchObjectAction(
+			objectActionId);
+
+		Assert.assertFalse(objectAction.isActive());
+
+		AssertUtils.assertFailure(
+			ObjectActionActiveException.class,
+			"Cannot activate object action if trigger is onAfterRootUpdate " +
+				"but object definition is not a root node",
+			() -> _objectActionLocalService.updateObjectAction(
+				objectAction.getExternalReferenceCode(),
+				objectAction.getObjectActionId(), true,
+				objectAction.getConditionExpression(),
+				objectAction.getDescription(),
+				objectAction.getErrorMessageMap(), objectAction.getLabelMap(),
+				objectAction.getName(),
+				objectAction.getObjectActionExecutorKey(),
+				objectAction.getObjectActionTriggerKey(),
+				objectAction.getParametersUnicodeProperties()));
```

Comment:
This test focuses on updating the Object Action. I think it is best if we add this scenario to the `testUpdateObjectAction()`. What do you think?